py/desispec/coaddition.py

```diff
@@ -297,10 +326,13 @@ def coadd_fibermap(fibermap, onetile=False):
         #         tfmap['LAST_'+k][i] = np.max(vals)
         #         tfmap['NUM_'+k][i] = np.unique(vals).size
 
+        #SJ: Not the correct formula for IVAR (should be inverse of sum of variances)
+        # because these are not for inverse variance weighted quantities.
+        # Those values are not used right now but should be fixed if used later
```

Comment:
Good catch!  even though we don't currently have these FIBER_RA/DEC_IVAR and DELTA_X/Y_IVAR columns, let's go ahead and fix this now as well.  I think the correction would be:
```python
tfmap[k][i] = 1/np.sum(1/fibermap[k][jj][compute_coadds])
```
Please double check that.  If one of the input ivars was 0 (infinite uncertainty), this will result in a numpy inf warning and an output ivar of 0, but I think that is what we want.  Do you agree, or otherwise what do you suggest?

Reply:
I guess if all IVARs are zero, I'd leave zero as the combined value (all invalid/masked). And if some are non-zero, I'd apply the formula to that subset. In an ideal world, the same subset with non-zero IVAR would also be combined (MEAN, STD) but by default one could make an assumption that those IVAR=0 cases would **not** be part of the `good_coadds` condition.

Reply:
I'll test it in case it already works with the formula before trying to implement something more sophisticated.

Reply:
I updated the comments and the formula to use the MEAN because I'm assuming that this would be error propagation on the mean values of those columns. I tested that one or more of the IVAR values being zero results in the output to be zero (with a dividing by zero warning).